runner/runner/handlers.py

```diff
@@ -58,6 +58,7 @@ def _run_task(task):
             detach=True,
             command=run_command,
             environment=variables,
+            network_mode="functionary-network",
```

Comment:
Is this even a valid value?  Maybe you were intending to set `network`? I acknowledge that this works as is somehow, but the [documentation](https://docker-py.readthedocs.io/en/stable/containers.html) makes me think that's a happy accident.

Either way this is problematic and we'll need to figure out how to handle this.  For now, I guess I'd say to make this an environment variable that we pass to the runner and then set it to this value in our docker-compose.yml.

Reply:
It should have just been `network="functionary-network"`. I'm honestly not sure why it lets me set `network_mode` to that value. Either way, the container that the runner spawns needs to be able to resolve the Minio container's hostname, so I figured I'd just add the container to the functionary network

Reply:
Right, I agree that this needs to be set to work in our dev environment, but you don't want to hard-code it in there.  It needs to be configurable so that it at least has a chance of working in some other setup.  In fact, we probably need both `network_mode` and `network` to be configurable and treated as optional.

Reply:
Apparently `network_mode` and `network` are mutually exclusive options according to the documentation. Or at least that's what I believe it means when it says "incompatible"

Reply:
Yeah, I know you can't set both, but there are situations where you might want to set one or the other (as in our dev setup case) and we should provide a way to do that.

Reply:
Fixed, although I don't check for true exclusive mutuality between network and network mode. I don't really see when we would want to modify the network mode. I figure that the task containers should always attach to the same network that the rest of the Functionary components are running on, as to not introduce a breaking configuration such that the task containers can no longer talk with Functionary's MinIO container